libs/api-interfaces/src/lib/api-interfaces.ts

```diff
@@ -15,6 +15,7 @@ export interface Volunteer {
   };
   active?: Boolean;
   registeredAt?: string;
+  recaptcha: string;
```

Comment:
should we also add a field for the privacy-checkbox?
e.g. `privacyAccepted: boolean;`

Reply:
yes, I think it would not be a big effort but could be required so that we know that someone accepted our data privacy